### DISCUSSION TITLE
Pushing to a new repo not possible as input required in the Command Log

### DISCUSSION THREAD

**Discussion by benfrain** [jesseduffield/lazygit#2155] [Q&A] [answered]
I have a new repo on GitLab I am trying to push to. I add a new remote and when I try and push I get the 'Pushing...' box in the centre but it is unable to complete as the Command Log area needs me to confirm I want to continue connecting.

Is there any way to resolve this inside Lazygit or do I need to do so on the CLI?

**Comment by mark2185** [comment]
Have you tried fetching the newly added remote first? Just `f` on it in the `Remotes` tab.

**Reply by benfrain** [reply]
No but thanks for the tip. I’ll try that next time.

**Comment by benfrain** [comment, marked as answer]
Turns out I was on a new machine and hadn't added a SHA key!

**Reply by mark2185** [reply]
Great! Glad it was solved :)
